foxglove_bridge_base/include/foxglove_bridge/websocket_server.hpp

```diff
@@ -38,6 +38,29 @@
     }                                                                                          \
   }
 
+namespace {
+
+constexpr uint32_t Integer(const std::string_view str) {
```

Comment:
this function should probably be named StringHash or something that actually explains what it does.

Is there any compile time guarantee that the strings we use it with actually hash to different values? I guess we would hope to get a compile error if it resulted in a switch with multiple identical `case`s?

edit: but I see we are not using switch..

Reply:
Renamed it and changed it back to a switch